src/operator/tensor/indexing_op.cc

```diff
@@ -13,10 +13,46 @@ DMLC_REGISTER_PARAMETER(TakeParam);
 DMLC_REGISTER_PARAMETER(OneHotParam);
 
 NNVM_REGISTER_OP(Embedding)
-.MXNET_DESCRIBE("Map integer index to vector representations (embeddings)."
-                " Those embeddings are learnable parameters. For a input of shape"
-                " (d1, ..., dK), the output shape is (d1, ..., dK, output_dim)."
-                " All the input values should be integers in the range [0, input_dim).")
+.describe(R"code(Maps integer indices to vector representations (embeddings).
+
+This operator maps words to real-valued vectors in a high-dimensional space,
```

Comment:
This can be expressed more clearly / compactly:

"This operator maps words to real-valued vectors in a high-dimensional space, called word embeddings."